cvmfs/cvmfs_config

```diff
@@ -1117,6 +1118,25 @@ cvmfs_bugreport() {
    echo "JIRA bug report to https://sft.its.cern.ch/jira/browse/CVM"
 }
 
+cvmfs_regularcheck() {
+  
+  if ! cvmfs_readconfig; then
+    echo "Failed to read CernVM-FS configuration"
+    return 1
+  fi
+
+  local repo_list
+  if [ "x$CVMFS_REPOSITORIES" != "x" ]; then
+    repo_list=`echo $CVMFS_REPOSITORIES | sed 's/,/ /g'`
+  fi
+
+  local repo
+  for repo in $repo_list
+  do
+     sudo /usr/bin/cvmfs_fsck ${CVMFS_CACHE_BASE}/${repo}
```

Comment:
The CVMFS_CACHE_BASE parameter can change from repository to repository.  You'd need to re-read the configuration before.